docker_daemon/check.py

```diff
@@ -425,6 +432,19 @@ def _get_tags(self, entity=None, tag_type=None):
                     ecs_tags = self.ecs_tags[entity_id]
                     tags.extend(ecs_tags)
 
+            # Add Mesos tags
+            if self.collect_mesos_tags and (tag_type is CONTAINER or tag_type is PERFORMANCE):
+                cont_id = entity.get("Id")
+                if cont_id is not None:
+                    try:
+                        cont_inspected = self.docker_client.inspect_container(cont_id)
+                        for env_var in cont_inspected['Config']['Env']:
+                            arr = env_var.split("=")
+                            if arr[0].upper() in MESOS_TAG_ENVS:
+                                tags.append("{}:{}".format(arr[0].lower(), arr[1]))
+                    except Exception as e:
+                        self.log.debug("Unable to inspect Docker container: %s", e)
```

Comment:
either this wraps only the inspect_container call or we should rewrite the message to something like `Unable to retrieve mesos tags`